actioncable/lib/action_cable/connection/subscriptions.rb

```diff
@@ -54,7 +54,7 @@ def identifiers
       end
 
       def unsubscribe_from_all
-        subscriptions.each { |id, channel| channel.unsubscribe_from_channel }
+        subscriptions.each { |id, channel| remove({ 'identifier' => id }) }
```

Comment:
Can we use `remove_subscription(channel)` here? This feels like unnecessary indirection (and I think I'd actually argue we *don't* want the log lines).